File: src/CorporateMessengerWithAgile/Application/Dto/Summary/ChatSummaryDto.cs
namespace Application.Dto.Summary
{
    public class ChatSummaryDto : BaseDto
    {
        public string Name { get; set; } = null!;
        public string Description { get; set; } = null!;
        public Guid? OwnerEmployeeId { get; set; }
        public Guid? OwnerTeamId { get; set; }
    }
}

File: src/CorporateMessengerWithAgile/ReactApp/reactapp.client/src/models/entity/ChatSummaryDto.ts
// src/models/entity/ChatSummaryDto.ts
import { z } from 'zod';
import { BaseDtoSchema } from './BaseDto';
import { GuidSchema } from '../Guid';

export const ChatSummaryDtoSchema = BaseDtoSchema.extend({
    name: z.string(),
    description: z.string(),
    ownerEmployeeId: GuidSchema.nullable(),
    ownerTeamId: GuidSchema.nullable()
});

export type ChatSummaryDto = z.infer<typeof ChatSummaryDtoSchema>;